cardano-node/Cardano/Node/Run.hs

```diff
@@ -18,15 +18,14 @@
 module Cardano.Node.Run (
       runNode
     ) where
-import           Cardano.Prelude hiding (ByteString, atomically, throwIO, wait)
+import           Cardano.Prelude hiding (ByteString, atomically, throwIO, trace, wait)
```

Comment:
Does `trace` need to be hidden here? I'm not seeing it used in the module.

Reply:
I see a parameter name being specified [here](https://github.com/input-output-hk/cardano-node/blob/5dd7682d91612bde0f8538506749c7663c5bebcf/cardano-node/Cardano/Node/Run.hs#L183), called `trace`. Perhaps `trace` was hidden in order to prevent name shadowing.

Reply:
an argument is also named "trace" so there was a warning.

Reply:
we should turn on `-Werror` once.

Reply:
@CodiePP: I agree on the note of `-Werror`. I quite like how it's done in `cardano-ledger`, where there's a `development` cabal flag for enabling/disabling the usage of `-Werror`:
```
flag development
    description: Disable `-Werror`
    default: False
    manual: True

...

  if (!flag(development))
    ghc-options:         -Werror

```